gradle.properties

```diff
@@ -5,9 +5,11 @@ org.gradle.jvmargs=-XX:MaxPermSize=512m -XX:+CMSClassUnloadingEnabled -XX:+CMSPe
 bundleInstallRoot = /apps/grabbit/install
 
 group = com.twcable.grabbit
-version = 7.1.5
+version = 7.1.6
 
 # Please keep alphabetical
+apache_httpclient_version = 4.5.4
+apache_httpcore_version = 4.4.8
```

Comment:
Since we are using the OkHttpClient already, it would be better to continue to use that than having another way to create connections using the Apache HttpClient.

Reply:
Ok will switch to that one